### PR TITLE
Add .yaml/.yml interchangeability

### PR DESCRIPTION
**Pull request by rebeccahhh**
##### SUMMARY

This is a simplified PR based on: #4773 
The changes are the same, the commits have been squashed and this branch is updated to devel.
related to: #4540 

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME

 - API

##### AWX VERSION

```
9.0.1
```

##### ADDITIONAL INFORMATION

For additional information see issue #4540 and PR #4773 by community member @Kandeh

### CODE REVIEW

File: awx/playbooks/project_update.yml [hunk 1]
```diff
@@ -126,41 +126,50 @@
...
           args:
             chdir: "{{project_path|quote}}/roles"
           register: galaxy_result
-          when: doesRequirementsExist.stat.exists
+          with_items: "{{ doesRequirementsExist.results }}"
+          when: item.stat.exists
           changed_when: "'was installed successfully' in galaxy_result.stdout"
           environment:
-            ANSIBLE_FORCE_COLOR: false
+            ANSIBLE_FORCE_COLOR: False
```
Comment (ryanpetrello) [inline awx/playbooks/project_update.yml]: We should revert this back, it's cleanup from a recent ansible lint PR that @Spredzy landed.

File: awx/playbooks/project_update.yml [hunk 2]
```diff
@@ -115,7 +115,7 @@
         - update_insights
 
     - name: Repository Version
-      debug: msg="Repository Version {{ scm_version }}"
+      debug: msg= "Repository Version {{ scm_version }}"
```
Comment (ryanpetrello) [inline awx/playbooks/project_update.yml]: We should revert this back, it's cleanup from a recent ansible lint PR that @Spredzy landed.

File: awx/playbooks/project_update.yml [hunk 3]
```diff
@@ -126,41 +126,50 @@
...
           args:
             chdir: "{{project_path|quote}}/collections"
           register: galaxy_collection_result
-          when: doesCollectionRequirementsExist.stat.exists
+          with_items: "{{ doesCollectionRequirementsExist.results }}"
+          when: item.stat.exists
           changed_when: "'Installing ' in galaxy_collection_result.stdout"
           environment:
-            ANSIBLE_FORCE_COLOR: false
+            ANSIBLE_FORCE_COLOR: False
```
Comment (ryanpetrello) [inline awx/playbooks/project_update.yml]: We should revert this back, it's cleanup from a recent ansible lint PR that @Spredzy landed.

**Review by ryanpetrello — APPROVED** [review]

File: awx/playbooks/project_update.yml (hunk 2 above)
Comment (Spredzy) [reply awx/playbooks/project_update.yml]: Agree on this one, not sure how it landed.

File: awx/playbooks/project_update.yml (hunk 1 above)
Comment (Spredzy) [reply awx/playbooks/project_update.yml]: Why here ?

File: awx/playbooks/project_update.yml (hunk 3 above)
Comment (Spredzy) [reply awx/playbooks/project_update.yml]: Why here ?

Comment (ryanpetrello) [reply awx/playbooks/project_update.yml]: i.e., `false` is fine

File: awx/playbooks/project_update.yml (hunk 1 above)
Comment (ryanpetrello) [reply awx/playbooks/project_update.yml]: i.e., `false` is fine

Comment (rebeccahhh) [reply awx/playbooks/project_update.yml]: will do

File: awx/playbooks/project_update.yml (hunk 2 above)
Comment (rebeccahhh) [reply awx/playbooks/project_update.yml]: clarification for me, take the space out, leave the space in?

Comment (ryanpetrello) [reply awx/playbooks/project_update.yml]: Whichever passes CI, specifically the style linter?  I suspect "no space" is the correct answer.

Comment (ryanpetrello) [reply awx/playbooks/project_update.yml]: `yamllint -s awx/playbooks/project_update.yml`

Comment (rebeccahhh) [reply awx/playbooks/project_update.yml]: ah, gotcha, I will fix

**Comment by rebeccahhh** [comment]
closing, completed work will be in an updated PR

**Comment by ryanpetrello** [comment]
replaced by: https://github.com/ansible/awx/pull/8312

**Pull request closed without merging**